sp/src/game/server/hl2/npc_antlion.cpp

```diff
@@ -3205,10 +3206,17 @@ bool CNPC_Antlion::HandleInteraction( int interactionType, void *data, CBaseComb
 		ApplyAbsVelocityImpulse( vecDir );
 		Flip();
 
-		// If the Combine soldier is a player, don't deal damage on top of flipping
-		if ( sender->IsPlayer() )
+		// If the player kicks the antlion, it receives concussion damage without bleeding - HEVcrab
+
```

Comment:
Last thing from me - remove the extra newline

Reply:
Done :)